scripts/linters/pre_commit_linter.py

```diff
@@ -91,7 +91,6 @@
 # another shard.  Currently we are not sharding the lint checks, so the
 # only shard is the `other` shard that contains all files.
 SHARDS = {
-    '1': ['core/tests/',],
```

Comment:
Odd, why was this here in the first place? Maybe @U8NWXD can verify.

Reply:
Actually, I have defined this shard only to get the errors from files of core/test simultaneously.